lib/util.js

```diff
@@ -1030,13 +1030,16 @@ function error(...args) {
 }
 
 function _errnoException(err, syscall, original) {
-  var name = errname(err);
+  if (typeof err !== 'number' || err >= 0 || !Number.isSafeInteger(err)) {
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'err',
+                               'negative number');
```

Comment:
Not particularly useful. The `ERR_INVALID_ARG_TYPE` actual prints out the *type* of the actual, not the value itself. So if I passed in `_errnoException(1)`, the error message would be `The "err" argument must be of type negative number. Received type number`, which is just *odd*. We *would* turn this into a `RangeError` if the `err` is a number and isn't negative, but that adds a second if-check that is not strictly necessary for what is *supposed* to be an internal API.